package.json

```diff
@@ -52,14 +52,24 @@
 	"license": "Apache-2.0",
 	"dependencies": {
 		"@babel/parser": "^7.2.2",
+		"@types/d3-array": "^2.0.0",
```

Comment:
My question is similar here: npm install --save @types/d3. This might be easier to maintain as we only update 1 file in package instead of many